### PR TITLE
FAD-4466 Add default meta data

### PR DESCRIPTION
**Pull request by jonambas**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,6 +3,28 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>SparkPost Tools</title>
```
Comment (jasonrhodes) [inline public/index.html]: We don't use helmet to manage per-route titles?

File: public/index.html [hunk 2]
```diff
@@ -3,6 +3,28 @@
...
+    <!-- These will not be overwritten by react-helmet -->
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="SparkPost">
+    <meta property="fb:admins" content="371333539709717">
+    <meta name="twitter:site" content="@SparkPost">
+    <meta name="twitter:creator" content="@SparkPost">
+    <meta name="twitter:card" content="summary_large_image">
+
+    <!-- These will be overwritten -->
+    <meta property="og:title" content="SparkPost Tools" data-react-helmet="true">
```
Comment (jasonrhodes) [inline public/index.html]: Not sure if the `="true"` is even necessary, I think the presence of an attribute by itself is interpreted as boolean true, but it doesn't hurt.

File: public/index.html [hunk 3]
```diff
@@ -3,6 +3,28 @@
...
+    <meta property="og:url" content="https://tools.sparkpost.com" data-react-helmet="true">
+    <meta property="og:image" content="https://tools.sparkpost.com/images/tools_og.png" data-react-helmet="true">
+    <meta property="og:description" content="Tools from the email experts at SparkPost make it easy to build and check email authentication settings for your domain." data-react-helmet="true">
+
+    <meta name="twitter:title" content="SparkPost Tools" data-react-helmet="true">
+    <meta name="twitter:image" content="https://tools.sparkpost.com/images/tools_twitter.png" data-react-helmet="true">
+    <meta name="twitter:description" content="Tools from the email experts at SparkPost make it easy to build and check email authentication settings for your domain." data-react-helmet="true">
```
Comment (jasonrhodes) [inline public/index.html]: Is there any advantage to setting twitter specific meta tags if we want it to use the same info in the og tag? I thought Twitter will use the og meta tags but I haven't looked at this stuff in a while so I'm not sure.

File: public/index.html (hunk 1 above)
Comment (jonambas) [inline public/index.html]: That's the one tag helmet overwrites automatically

File: public/index.html (hunk 3 above)
Comment (jonambas) [inline public/index.html]: Possibly but recommended dimensions/aspect ratios are different I think?

Comment (jonambas) [inline public/index.html]: Twitter has a much smaller file size requirment

**Pull request merged**